packages/ketcher-core/src/application/editor/data/monomers.ket

```diff
@@ -169036,7 +169066,12 @@
     "name": "MOE(T)P",
     "class": "RNA",
     "idtAliases": {
-      "base": "2MOErT"
+      "base": "2MOErT",
+      "modifications": {
+        "endpoint5": "52MOErT",
+        "internal": "i2MOErT",
+        "endpoint3": "32MOErT"
+      }
```

Comment:
I remember how we decided to not fill field 'modifications' it base rules are applicable (5/i/3 + base). Did we change this decision?

Reply:
Reverted, added logic to create modifications from base for presets